src/react/ISV/components/ISVCapacityFormSection.tsx

```diff
@@ -0,0 +1,138 @@
+import { FormGroup, FormSection, Stack } from '@scality/core-ui';
+import { Input, Select } from '@scality/core-ui/dist/next';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { useEffect } from 'react';
+import { useXcoreRuntimeConfig } from '../../next-architecture/ui/ConfigProvider';
+import { useShellHooks } from '@scality/module-federation';
+import { ListItem } from './ISVApplyActions';
+import { unitChoices } from '../constants';
+import { useCapacityUnit } from '../hooks/useCapacityUnit';
+
+type XCoreConfig = {
+  spec: {
+    selfConfiguration: {
+      url: string;
+      url_alertmanager: string;
+      url_prometheus: string;
+      url_grafana: string;
+    };
+  };
+};
+type ClusterCapacityStatus = 'idle' | 'loading' | 'success' | 'error';
+type UseClusterCapacityHooks = (
+  xCoreConfig: XCoreConfig,
+  getToken: () => Promise<string | null>,
+) => {
+  clusterCapacity: number;
+  clusterCapacityStatus: ClusterCapacityStatus;
+};
+
+const CapacityTooltip = () => (
+  <ul>
+    <ListItem>
+      Set your ARTESCA storage capacity limit to be monitored by Veeam (via
+      Smart Object Storage API).
+    </ListItem>
+    <ListItem>
+      Keep in mind, going over this limit has no effect on ARTESCA itself, but
+      it does trigger a warning in the Veeam UI and can potentially stop backup
+      activities.
+    </ListItem>
+  </ul>
+);
+
+export const CapacityFormWithXcore = ({
+  useClusterCapacity,
+  index,
+}: {
+  useClusterCapacity: UseClusterCapacityHooks;
+  index: number;
+}) => {
+  const { useAuth } = useShellHooks();
+  const { getToken } = useAuth();
+  const xCoreConfig = useXcoreRuntimeConfig();
+  const { clusterCapacity, clusterCapacityStatus } = useClusterCapacity(
+    xCoreConfig,
+    getToken,
+  );
+  const { setValue } = useFormContext();
+  const { capacityValue, capacityUnit } = useCapacityUnit(
+    clusterCapacityStatus === 'success'
+      ? clusterCapacity * (0.8 / index + 1)
```

Comment:
Shouldn't it be the total number of buckets here instead of an index ?
```suggestion
      ? clusterCapacity * (0.8 / (index + 1))
```